package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/auth-client",
-  "version": "2.63.0",
+  "version": "2.63.1",
```

Comment:
Since this is a feature and not a bug fix, this version should be `2.64.0`.  https://semver.org/

Reply:
I have updated this to be 2.64.0 in most recent commit